cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -300,6 +302,50 @@ char* gdv_fn_dec_to_string(int64_t context, int64_t x_high, uint64_t x_low,
     return val;                                                                      \
   }
 
+#define GDV_FN_TO_CHAR(IN_TYPE, ARROW_TYPE)                                         \
+  /* Converts expression to a string using the format specified in format */                                                                                  \
+  GANDIVA_EXPORT                                                                         \
+  const char* gdv_fn_to_char_##IN_TYPE##_int64(int64_t context, gdv_##IN_TYPE value, \
+                                                   int64_t len, const char* pattern, int32_t * out_len) {     \
```

Comment:
You are not using pattern at all in your code. Doesn't look like it should be working.

Reply:
You're right. I lost something among my commits and merges. Looking into it now.